### PR TITLE
a couple of little fixings

### PR DESCRIPTION
**Pull request by juanky201271**
I changed 'grey' to 'gray'. (I believe 'grey' doesn't exists).
I changed: onPress={() => !disabled && onPress()}>
to: onPress={() => !disabled && onPress && onPress()}>
because of onPress have a null or undefined value. This way both Buttons have the same event behavior of OnPress.

### CODE REVIEW

File: components/Components.tsx [hunk 1]
```diff
@@ -78,14 +78,14 @@ export const PrimaryButton: React.FunctionComponent<any> = ({title, disabled, on
...
-        backgroundColor: disabled ? 'grey' : colors.primary,
+        backgroundColor: disabled ? 'gray' : colors.primary,
         padding: 10,
         paddingLeft: 20,
         paddingRight: 20,
         borderRadius: 10,
         width: '40%',
       }}
-      onPress={() => !disabled && onPress()}>
+      onPress={() => !disabled && onPress && onPress()}>
```
Comment (zancas) [inline components/Components.tsx:88]: What's the `onPress` variable?   It seems confusing to have a variable and a function with the same name?   Or are they the same object, just being references and then called?

Comment (juanky201271) [reply components/Components.tsx:88]: This is a curious technique of JS, means: if It has a valid value (not null, not undefined and not 0) inside of onPress then asign onPress function to onPress event of the button. But my goal here is leave the two kind of buttons with the same code, if this is possible.

Comment (juanky201271) [reply components/Components.tsx:88]: You can review the code between PrimaryButton and SecondaryButton, with the modification onPress is identical in both. I called this coherence.

Comment (zancas) [reply components/Components.tsx:88]: When I see a situation like `PrimaryButton` and `SecondaryButton` I think:

"Time to define a a helper function to DRY out this code!!!"

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

**Comment by juanky201271** [comment]
I did the Button refactor, I added a new commit. (and verified, jejejeje)

**Comment by zancas** [comment]
The parametrized "Button" defined in a single place seems like a very nice refactor to me!

I'd prefer to have another reviewer look over the code, since I don't have a background in JSX...

**Comment by zancas** [comment]
Is it now possible to delete `PrimaryButton` and `SecondaryButton`?  Since they are now captured as differently parametrized `Buttons`?

**Review by Gunnar-Stunnar — APPROVED** [review]
Looks good! Good eye on gray, I was not aware of the primary button vs secondary button

**Comment by zancas** [comment]
@Gunnar-Stunnar do you want to merge this?

**Comment by juanky201271** [comment]
> Is it now possible to delete `PrimaryButton` and `SecondaryButton`? Since they are now captured as differently parametrized `Buttons`?

Yep, I can do that.

**Pull request merged**